docs/Secure-Coding-Guide-for-Python/CWE-707/CWE-175/example03.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: OpenSSF project contributors
+# SPDX-License-Identifier: MIT
+""" Non-compliant Code Example """
+import locale
+ORIGINAL_NUMBER = 12.345  # This will read as 12,345 in German
+ 
+def compare_number(number):
+    input_number = locale.atof(input("Enter a number: "))
+    # Test if inputted number equals current number
+    return number == input_number
+
+print(f"Locale is {locale.getlocale()}")
```

Comment:
Getting this from Flake:
expected 2 blank lines after class or function definition, found 1Flake8(E305)
(function) def print(
    *values: object,
    sep: str | None = " ",
    end: str | None = "\n",
    file: SupportsWrite[str] | None = None,
    flush: Literal[False] = False
) -> None
Prints the values to a stream, or to sys.stdout by default.

sep
  string inserted between values, default a space.